architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -432,4 +432,39 @@ to run:
 `oc delete istag/ruby:latest`
 ====
 
+[[insecure-registries]]
 
+=== Importing images from insecure registries
+An image stream can be configured to import tag and image metadata from an image
+repository that is signed with a self-signed certificate. To do that, add
+`*openshift.io/image.insecureRepository*` annotation set to `*true*`. This setting
+will bypass certificate validation when connecting to registry.
+
+====
+
+[source,json]
+----
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "ruby",
+    "annotations": {
+      "openshift.io/image.insecureRepository": "true"
+    }
+  },
+  "spec": {
+    "dockerImageRepository": "my.repo.com:5000/myimage"
+  }
+}
+----
+====
```

Comment:
@soltysh Any chance we can have a yaml example instead? That's what we're trying to aim for in the docs.